RELEASE_NOTES.md

```diff
@@ -1,9 +1,16 @@
 Release Notes
 =============
 
-v1.0.0-beta.x.y
+v1.0.0-beta.2.2
 ---------------
 
+_This release introduces new features, and remains binary and source
+compatible for hosts, and source compatible for managers._
+
+_The addition of a new virtual method,
+ `LoggerInterface.isSeverityLogged`, introduces a binary incompatibility
+ for upgrading managers._
```

Comment:
[fix] I think we have to be more strict than this, because on Windows apparently any new virtual method can cause ABI breaks.

So I think we remain source compatible for both hosts and managers, but that's it.

Reply:
Is this new information since we discussed on Friday, when we thought only managers were a problem?
In any case, i'm fine with being more careful than we need to be, updated the message to be broader.
https://github.com/OpenAssetIO/OpenAssetIO/pull/1334/commits/1308cf32066ca64fbd797cde3294845762211faa